README.md

```diff
@@ -133,25 +133,52 @@ the default servient.
 If you want to use node-wot as a library in your Node.js application, you can use npm to install the node-wot packages that you need. To do so, `cd` inside you application folder, and run:
 
 ```
-npm i @node-wot/core @node-wot/binding-coap --save
+npm i @node-wot/core @node-wot/binding-http --save
 ```
 
-Now, you can implement your node-wot entry point, e.g., `main.js` as follows:
+Now, you can implement your a thing as follows:
```

Comment:
either 
* "your thing" OR
* "a thing" without your

Reply:
Oh didnt see that. It should be fixed now. Thanks!